frontend/webpack.config.js

```diff
@@ -20,7 +20,8 @@ module.exports = {
   resolve: {
     extensions: ['.js', '.jsx'],
     alias: {
-      modules: `${__dirname}/node_modules`
+      modules: `${__dirname}/node_modules`,
+      app: `${__dirname}/src`
```

Comment:
Where the alias been used? Maybe should be in `webpack.config.prod.js` (https://github.com/worknenjoy/gitpay/blob/master/frontend/webpack.config.prod.js)?